test/expect-tests/dune_patch/dune_patch_tests.ml

```diff
@@ -0,0 +1,46 @@
+open Stdune
+
+let () = Dune_tests_common.init ()
+
+let basic =
```

Comment:
You should add tests testing the entire action here. This is where it will be useful to expose `Spec.action` without all the scaffolding.

Reply:
OK, I'll write some tests.